examples/lock-app/nrfconnect/main/AppTask.cpp

```diff
@@ -385,6 +378,22 @@ void AppTask::StartBLEAdvertisementHandler(AppEvent * aEvent)
     if (aEvent->ButtonEvent.PinNo != BLE_ADVERTISEMENT_START_BUTTON)
         return;
 
+    if (!sNFC.GetTagState())
+    {
+        if (!GetAppTask().StartNFCTag())
```

Comment:
That is most likely fine, though you can consider sth like:

`if (!(GetAppTask().StartNFCTag() < 0))` in order to understand what is returned just by looking at the code.